packages/kbn-test/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@kbn/test",
+  "main": "./target/index.js",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "build": "babel src --out-dir target",
+    "kbn:bootstrap": "yarn build"
```

Comment:
:loudspeaker: (aka announcement or super optional hint): you can also define `"kbn:watch": "yarn build --watch"` and run `yarn kbn watch -i @kbn/test` during development :)

Reply:
neat!